### ISSUE TITLE
Mac 2.0.9 regression: Open-then-play no longer plays

### ISSUE THREAD

**Issue by TheloTheGreat** [RenderHeads/UnityPlugin-AVProVideo#664]
On Mac AVPro 2.0.9, but not on Windows, calling MediaPlayer.OpenMedia followed by MediaPlayer.Play no longer actually plays the video. Instead the video stays stopped. (2.0.8 did not have that problem.)

The video _does_ play if you call Seek before Play, however.

Repro: In Demo_MediaPlayer on Mac, add this to MediaPlayerUI.Update:

```
if (Input.GetKeyDown(KeyCode.A)) {
	_mediaPlayer.OpenMedia(MediaPathType.RelativeToProjectFolder, "Assets/StreamingAssets/AVProVideoSamples/BigBuckBunny-360p30-H264.mp4", autoPlay: false);
	_mediaPlayer.Control.Play();
}
if (Input.GetKeyDown(KeyCode.S)) {
	_mediaPlayer.OpenMedia(MediaPathType.RelativeToProjectFolder, "Assets/StreamingAssets/AVProVideoSamples/BigBuckBunny-360p30-H264.mp4", autoPlay: false);
	_mediaPlayer.Control.Seek(1.0);
	_mediaPlayer.Control.Play();
}
```

Then press the A key.
Expected result: The video starts playing from the beginning.
Actual result: The video gets loaded, but stays stopped at the beginning.

Pressing the S key, which involves a Seek operation, does play the video as expected.

**Comment by MorrisRH** [comment]
A fix will be in the next release.

**Comment by TheloTheGreat** [comment]
Fix confirmed on 2.0.10, thanks.
